code/src/components/Checkbox.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import "./form.css";
+import "./checkbox.css";
+
+const Checkbox = ({ isBoxChecked, onCheckBoxChange }) => {
+
+  return (
+    <section className="section-container">
+      <h2 tabIndex="0">I would like a price estimation for:</h2>
+      <label htmlFor="moving">
+        <input
+          tabIndex="0"
+          type="checkbox"
+          id="moving"
+          onChange={() => onCheckBoxChange("moving")}
+          checked={isBoxChecked.includes("moving")}
+        >
+        </input>
+        Moving
+      </label>
+      <label htmlFor="packing">
+        <input
+          tabIndex="0"
+          type="checkbox"
+          id="packing"
+          onChange={() => onCheckBoxChange("packing")}
+          checked={isBoxChecked.includes("packing")}
+        >
+        </input>
+        Packing
+      </label>
+      <label htmlFor="cleaning">
+        <input
+          tabIndex="0"
+          type="checkbox"
+          id="cleaning"
+          // onChange={(event) => onCheckBoxChange(event.target.checked)} - what is the difference?
```

Comment:
This `onChange` event will update `checkBox` state property from `<Form />` component with value `true` or `false` - depends on if you uncheck or check this input.
Line below, `onChange={() => onCheckBoxChange("cleaning")}` will update state property with string value `cleaning`. I assume you want the second line to be executed rather than the commented one :)